torch/onnx/symbolic_opset9.py

```diff
@@ -1539,6 +1539,11 @@ def log1p(g, self):
     return log(g, add(g, sym_help._if_scalar_type_as(g, torch.ones(1), self), self))
 
 
+def log10(g, self):
+    log_div = g.op("Log", g.op("Constant", value_t=torch.tensor(10.0)))
```

Comment:
`log_e{10.0}` could be added as a constant, similar to how existing symbolic log2 works. what do you think?

Reply:
Modified to match the format